workshops/migrations/0088_auto_20160530_1029.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2016-05-30 15:29
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.db.models import F, Value as V
+from django.db.models.functions import Concat, Lower
+from django.template.defaultfilters import slugify
+
+
+class Migration(migrations.Migration):
+
+    def slugify(apps, schema_editor):
+        """Generate slug from date and venue."""
+        Event = apps.get_model('workshops', 'Event')
+        for event in Event.objects.filter(slug__isnull=True):
+            event.slug = slugify(str(event.start) + '-' + event.venue)
```

Comment:
Have you run into troubles with the current approach? If yes, then please add such a loop.